programs/keeper/clickhouse-keeper.cpp

```diff
@@ -1,6 +1,35 @@
+#include "config_tools.h"
+#include <Common/StringUtils/StringUtils.h>
+
+#include <vector>
+
+
 int mainEntryClickHouseKeeper(int argc, char ** argv);
 
+#if ENABLE_CLICKHOUSE_KEEPER_CLIENT
+int mainEntryClickHouseKeeperClient(int argc, char ** argv);
+#endif
+
 int main(int argc_, char ** argv_)
 {
+#if ENABLE_CLICKHOUSE_KEEPER_CLIENT
+    std::vector<char *> argv(argv_, argv_ + argc_);
+
+    if (argv.size() >= 2)
+    {
+        auto first_arg = argv.begin() + 1;
+
+        /// 'clickhouse-keeper --client ...' and 'clickhouse-keeper client ...' are OK
+        if (strcmp(*first_arg, "--client") == 0 || strcmp(*first_arg, "client") == 0)
+        {
+            argv.erase(first_arg);
+            return mainEntryClickHouseKeeperClient(static_cast<int>(argv.size()), argv.data());
+        }
+    }
+
+    if (!argv.empty() && (strcmp(argv[0], "clickhouse-keeper-client") == 0 || endsWith(argv[0], "/clickhouse-keeper-client")))
```

Comment:
I guess you need to create symlink from standalone keeper for this
something like

```
 add_custom_target (clickhouse-server ALL COMMAND ${CMAKE_COMMAND} -E create_symlink clickhouse clickhouse-server DEPENDS clickhouse)
 ```